src/Components/Web/src/Forms/InputSelect.cs

```diff
@@ -16,6 +16,12 @@ public class InputSelect<TValue> : InputBase<TValue>
         /// </summary>
         [Parameter] public RenderFragment? ChildContent { get; set; }
 
+        /// <summary>
+        /// Gets or sets the associated <see cref="ElementReference"/>
+        /// <para>This value is used to gain access to helper methods on the input elment.</para>
+        /// </summary>
+        public ElementReference? InputElement { get; protected set; }
```

Comment:
I personally think `SelectElement` would be better as to not give the impression that it's referring to an `<input>` element.

@pranavkm, do you have a strong opinion on this?

Reply:
Changed